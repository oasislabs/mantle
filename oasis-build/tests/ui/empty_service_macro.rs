fn main() {
    oasis_std::service!();
}

