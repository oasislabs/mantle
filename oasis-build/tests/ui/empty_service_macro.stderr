error: unexpected end of macro invocation
 --> $DIR/empty_service_macro.rs:2:5
  |
2 |     oasis_std::service!();
  |     ^^^^^^^^^^^^^^^^^^^^^^ missing tokens in macro arguments

error: aborting due to previous error

